doc/exploit/stream_connectors.rst

```diff
@@ -202,10 +203,14 @@ Here is an example showing the possibilities of the ``parse_perfdata`` function.
 
 .. code-block:: lua
 
-  local perf = broker.parse_perfdata(" 'one value'=2s;3;5;0;9 'a b c'=3.14KB;0.8;1;0;10")
+  local perf, err_str = broker.parse_perfdata(" 'one value'=2s;3;5;0;9 'a b c'=3.14KB;0.8;1;0;10")
 
-  for i,v in pairs(perf) do
-    print(i .. " => " .. tostring(v))
+  if err_str == nil then
```

Comment:
A suggestion, should not we  check the perf value and if it is nil, then we read the error value ? I see it more understandable as an example.